src/index.ts

```diff
@@ -361,23 +361,52 @@ function getAlbService(albName: string, environment: string): string {
     .replace('-alb', '')
 }
 
-function transformALBLog(data: S3LogRecord, envVars: EnvVars): TransformationResult {
+function extractAlbLogLineTime(log: string): number | undefined {
+  const regex = /(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+Z)/
+  const extractedTime = regexTimeFromLog(regex, log)
+  console.log(extractedTime)
```

Comment:
Is there any reason this is logged for every log line along with exceptional cases in the `transformALBLog` method?

Reply:
No, thank you so much that shouldn't be there and I've removed it, I mustn't have done my usual search for `console.log` after debugging and pushing 😓